Common/File/FileUtil.cpp

```diff
@@ -137,7 +137,12 @@ FILE *OpenCFile(const Path &path, const char *mode) {
 				INFO_LOG(COMMON, "Opening '%s' for write failed", path.ToString().c_str());
 				return nullptr;
 			}
-			return fdopen(descriptor, "wb");
+			FILE *f = fdopen(descriptor, "wb");
+			if (!strcmp(mode, "at") || !strcmp(mode, "a")) {
+				// Append mode.
```

Comment:
Well, append is supposed to *always* write to the end of the file (i.e. regardless of any seek calls later.)  But I suppose this API will likely never use it that way.

-[Unknown]

Reply:
Yeah, it's true that the semantics are somewhat off. But yeah... can't be bothered.